platform-hub-auth-proxy/README.md

```diff
@@ -4,8 +4,6 @@ Uses [keycloak-proxy](https://github.com/gambol99/keycloak-proxy)
 
 The `local` folder contains scripts to manage a local `keycloak-proxy` instance in a Docker container.
 
-**IMPORTANT:** to allow the Docker container to talk to the API running on your host: set up an alias on your local loopback adapter using `sudo ifconfig lo0 alias 10.200.10.1/24` (needed on every restart of your computer).
```

Comment:
Can you maybe include a note for the minimum Docker version required on OSX?

Reply:
Good point!